index.html

```diff
@@ -1,13 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <div class="container"></div>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Vanilla Todo</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="container">
+    <h2> 대기중(
+      <span id="pendingTasksNumber">0</span>
+      )
+    </h2>
+    <ul id="todosPendingList">
+
+    </ul>
+  
+    <h2> 완료됨(
+      <span id="finishedTasksNumber">0</span>
+      )
+    </h2>
+    <ul id = "todosFinishedList">
+    
+        
+    </ul>
+  
+    <form class="inputNewTasks">
+      <input id="todoInput" type="text" size="20" placeholder="새로운 할 일 추가">
```

Comment:
input에 size 제한을 두신 특별한 이유가 있을까요?? 내용이 컨테이너를 벗어나는것을 막기 위해 그러셨던거라면 `CSS overflow` 속성을통해 처리해주시는 편이 더 좋을 것 같아요~ [참고 ](https://developer.mozilla.org/ko/docs/Web/CSS/overflow)

```suggestion
      <input id="todoInput" type="text" placeholder="새로운 할 일 추가">
```